setup.py

```diff
@@ -37,12 +37,10 @@ def find_version(*file_paths):
         'License :: Apache 2.0',
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
-        'Programming Language :: Python :: 3.8'
+        'Programming Language :: Python :: 3.8',
     ],
     keywords='pytorch layer function approximation spline',
     packages=find_packages(exclude=['build', 'dist', 'docs', 'tests']),
     python_requires='>=3.6',
-    install_requires=[
-        'torch >= 1.4',
-    ],
+    install_requires=['torch >= 1.4'],
```

Comment:
Update coherently with `requirements.txt`

Reply:
The packages in `requirements.txt` describes the setup used for testing.
The package doesn't include tests when you install it and so it doesn't require numpy, scipy and pytest to work.